### Controller Parameters
controller: {

x_dot_P: 0.5,
x_dot_I: 0.0,
x_dot_D: 0.13,

y_dot_P: 0.5,
y_dot_I: 0.0,
y_dot_D: 0.13,

z_dot_P: 0.6,
z_dot_I: 0.25,
z_dot_D: 0.1,

north_P: 1.0,
north_I: 0.05, #1.0, #0.1,
north_D: 0.6,

east_P: 1.0,
east_I: 0.05, #1.0, #0.1,
east_D: 0.6,

down_P: 1.0,
down_I: 0.0,
down_D: 0.0,

psi_P: 2.0,
psi_I: 0.0,
psi_D: 0.0,

Kff_x: 1.0,
Kff_y: 1.0,
Kff_u: 0.07,
Kff_v: 0.07,
ramp_down_gain: 0.90,
conditional_integrator_threshold: 0.5,

tau: 0.05,

equilibrium_throttle: 0.5,

max_roll: 0.196,
max_pitch: 0.196,
max_yaw_rate: 0.785,
max_throttle: 0.85,
max_n_dot: 3.0,
max_e_dot: 3.0,
max_d_dot: 0.5,

# Below this altitude, zero roll, pitch, yawrate will be commanded
min_altitude: 0.0, #this needs to be zero for landing

}
